azure-communication-ui/azure-communication-ui/src/main/res/layout/azure_communication_ui_fragment_setup.xml

```diff
@@ -170,7 +170,7 @@
 
         <com.microsoft.fluentui.widget.Button
             android:id="@+id/azure_communication_ui_setup_join_call_button"
-            style="@style/AzureCommunicationUICalling.Button"
```

Comment:
even though it's effectively an alias for `Widget.FluentUI.Button` at the moment,  it's still more flexible to retain this style definition so then if theming requirements ever change in future we only need to modify `azure_communication_ui_styles.xml` in one location